cloudify_aws/common/utils.py

```diff
@@ -574,3 +581,175 @@ def raise_on_substring(iface,
         if any(substring in message for substring in substrings):
             raise raisable(message)
         return {}
+
+
+def with_rest_client(func):
+    """
+    :param func: This is a class for the starlingx resource need to be
+    invoked
+    :return: a wrapper object encapsulating the invoked function
+    """
+
+    def wrapper_inner(*args, **kwargs):
+        kwargs['rest_client'] = get_rest_client()
+        return func(*args, **kwargs)
+    return wrapper_inner
+
+
+@with_rest_client
+def create_deployments(group_id,
```

Comment:
same, if its common functions we are going to use a lot let's move them to utilities sdk and maintain them there.